blog/2025/rippled-2.4.0.md

```diff
@@ -0,0 +1,171 @@
+- Allow for a staged migration with minimal impact on validators and node operators.
+
+Keeping these goals in mind, a new subdomain, **unl.xrplf.org**, has been introduced alongside the existing https://vl.xrplf.org/. This approach allows for a staged migration without modifying the key for the current UNL list. Eventually, vl.xrplf.org will be fully deprecated and replaced by unl.xrplf.org.
+
+In `rippled` release 2.4.0, the validators-example.txt has been updated to include the new public key published by the new XRPL Foundation.
+
+Further information on the transition will be shared by the XRPL Foundation in the near future.
+
+
+## Action Required
+
+If you run an XRP Ledger server, upgrade to version 2.4.0 as soon as possible to ensure service continuity.
+
+Additionally, new amendments are now open for voting according to the XRP Ledger's [amendment process](../../docs/concepts/networks-and-servers/amendments.md), which enables protocol changes following two weeks of >80% support from trusted validators. The exact time that protocol changes take effect depends on the voting decisions of the decentralized network.
+
+## Install / Upgrade
+
+**TODO: Update package and SHA-256**
+
+On supported platforms, see the [instructions on installing or updating `rippled`](../../docs/infrastructure/installation/index.md).
+
+| Package | SHA-256 |
+|:--------|:--------|
+| [RPM for Red Hat / CentOS (x86-64)](https://repos.ripple.com/repos/rippled-rpm/stable/rippled-2.3.0-1.el7.x86_64.rpm) | `fb74f401e5ba121bbc37e6188aa064488ad78ffef549a1e19bc8b71316d08031` |
+| [DEB for Ubuntu / Debian (x86-64)](https://repos.ripple.com/repos/rippled-deb/pool/stable/rippled_2.3.0-1_amd64.deb) | `5d616d53218b47a2f0803c1d37d410f72d19b57cdb9cabdf77b1cf0134cce3ca` |
+| [Portable Builds (Linux x86-64)](https://github.com/XRPLF/rippled-portable-builds) | (Use signature verification) |
+
+For other platforms, please [build from source](https://github.com/XRPLF/rippled/blob/master/BUILD.md). The most recent commit in the git log should be the change setting the version:
+
+```text
+commit f64cf9187affd69650907d0d92e097eb29693945
+Author: Elliot Lee <github.public@intelliot.com>
+Date:   Mon Nov 25 12:27:17 2024 -0800
+
+    Set version to 2.3.0
+```
+
+
+## Full Changelog
+
+### Amendments
+
+The following amendments are open for voting with this release:
+
+
+- **XLS-46 DynamicNFT** - Adds the ability to update the URI of `NFToken` objects. ([#5048](https://github.com/XRPLF/rippled/pull/5048))
+- **XLS-80 Permissioned Domains** - Adds Permissioned Domains, which act as part of broader systems on the XRP Ledger to restrict access to satisfy compliance rules. ([#5161](https://github.com/XRPLF/rippled/pull/5161))
+- **XLS-77 Deep Freeze** - Adds the ability to deep freeze trust lines, enabling token issuers to block the transfer of assets for holders who have been deep frozen. ([#5187](https://github.com/XRPLF/rippled/pull/5187))
```

Comment:
Although I do like mentioning the XLS numbers, I don't know if putting it here in bold is right, since some of the items have only the exact amendment name in bold and others have the XLS number and not-exactly the amendment name (which won't have spaces). Maybe something like this instead:

> **DeepFreeze** (XLS-77) - Adds the...